mw-webapp/src/logic/wayPage/WayPage.tsx

```diff
@@ -296,11 +295,11 @@ export const WayPage = (props: WayPageProps) => {
       : "Not finished yet...";
 
     return (
-      <Tooltip content={tooltipContent}>
-        <div
-          key={singleGoalMetric.metricUuid}
-          className={styles.singularMetric}
-        >
+      <div
```

Comment:
Do we need both div and horizontal container? Not much wrappers?

Reply:
Yes, its need

Reply:
Why